plugins/tabletools/plugin.js

```diff
@@ -1896,6 +1896,7 @@
 	 *
 	 * @since 4.7.0
 	 * @cfg {Boolean} [tableImprovements=true]
+	 * @member CKEDITOR.config
```

Comment:
Please revert this change as it will be deprecated by a soon to be merged table selection plugin extraction.